test/index.test.ts

```diff
@@ -267,16 +267,25 @@ export default (test, assert) => {
         await gretch(`http://127.0.0.1:${port}`, {
           timeout: 50000,
           hooks: {
-            before (request) {
+            before (request, opts) {
+              assert(request.url)
+              assert(opts.timeout)
               hooks++
             },
-            after ({ status }) {
-              hooks++
-            }
+            after: [
+              (response, opts) => {
+                assert(response.status)
+                assert(opts.timeout)
+                hooks++
+              },
+              () => {
+                hooks++
+              }
+            ]
           }
         }).json()
 
-        assert(hooks === 2)
+        assert(hooks === 3)
```

Comment:
Did we get rid of the old test? If we're keeping backward-compatibility I'd love to keep the old test so that we have tests for both usages.

Reply:
Good Q! The `before` hook is the same syntax as before, `after` has been updated to be an array. So this is covering both usages.